io.github.igorlogius.scr2ppm.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.igorlogius.scr2ppm
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: scr2ppm-wrapper
+rename-icon: scr2ppm
+rename-desktop-file: scr2ppm.desktop
+rename-appdata-file: scr2ppm.metainfo.xml
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=xdg-pictures:create
+modules:
+  - name: scr2ppm
+    buildsystem: simple
+    build-commands:
+      - cc scr2ppm.c -lX11 -lxcb -lXau -lXdmcp -o scr2ppm
+      - install -D scr2ppm /app/bin/scr2ppm
+      - install -D scr2ppm-wrapper /app/bin/scr2ppm-wrapper
+      - install -D scr2ppm.desktop /app/share/applications/scr2ppm.desktop
+      - install -D scr2ppm.metainfo.xml /app/share/metainfo/scr2ppm.metainfo.xml
+      - install -D scr2ppm.png /app/share/icons/hicolor/128x128/apps/scr2ppm.png
+    sources:
+      - type: git
+        url: https://github.com/igorlogius/scr2ppm.git
+        branch: main
+        commit: 429dcc663ff40aaeb5449444ec73b35a0715e48b
```

Comment:
only a commit, because branch changes, that mean if there is an update to the branch, the build will fail.

Reply:
ah, ok